src/frontends/pytorch/src/op/norm.cpp

```diff
@@ -326,6 +327,62 @@ OutputVector translate_frobenius_norm(const NodeContext& context) {
     return {result};
 }
 
+OutputVector translate_rms_norm(const NodeContext& context) {
+    // Tensor = aten::rms_norm(%input_data.1, %2, %4, %3)
+    num_inputs_check(context, 2, 4);
+    auto x = context.get_input(0);
+    auto normalized_shape = context.get_input(1);
+    Output<Node> eps;
+    if (!context.input_is_none(3)) {
+        eps = context.get_input(3);
+        if (eps.get_element_type().is_dynamic() || eps.get_element_type() != x.get_element_type())
+            eps = context.mark_node(std::make_shared<v1::ConvertLike>(eps, x));
+    } else {
+        switch (x.get_element_type()) {
+        case element::bf16:
+            eps = context.mark_node(v0::Constant::create(ov::element::bf16, {}, {0.0078125}));
+            break;
+        case element::f16:
+            eps = context.mark_node(v0::Constant::create(ov::element::f16, {}, {0.0009765625}));
+            break;
+        case element::f64:
+            eps = context.mark_node(v0::Constant::create(ov::element::f64, {}, {2.220446049250313e-16}));
+            break;
+        case element::f32:
+            eps = context.mark_node(v0::Constant::create(ov::element::f32, {}, {1.1920928955078125e-07}));
+            break;
+        default:
+            eps = context.mark_node(v0::Constant::create(ov::element::f32, {}, {1.1920928955078125e-07}));
```

Comment:
please leave a comment where you get this constants

Reply:
Yeah, it is actual accessible with numerical_limits, fixed